spec/features/secure_routes_spec.rb

```diff
@@ -0,0 +1,41 @@
+# Copyright 2011-2015, The Trustees of Indiana University and Northwestern
+#   University.  Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+#   under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+#   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+#   specific language governing permissions and limitations under the License.
+# ---  END LICENSE_HEADER BLOCK  ---
+
+require 'rails_helper'
+
+describe "Secure Routes" do
```

Comment:
I looked at route specs, but that would require reaching down into the engines we're mounting. I also couldn't figure out how to write routing tests that take advantage of the state/identity of the logged in user. There's no `login_as` available to routing specs.